english/light-fire/fire.md

```diff
@@ -8,6 +8,8 @@ As the fire grows, a sweet aroma fills the room and the ceiling becomes ever so
 
 The walls around you begin to melt, globs of flaming marshmallow falling all around you. What do you do?
 
+As you somehow avoid the globs of marshmallow using your reflexes, you notice a big enough opening in the melted walls for you to squeeze through.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.